configs/BLADE_PRO_H7/config.h

```diff
@@ -0,0 +1,153 @@
+#define MOTOR8_PIN           PD13
+
+#define SERVO1_PIN           PE5
+#define SERVO2_PIN           PE6
+#define SERVO3_PIN           PD15
+#define SERVO4_PIN           PD14
+
+#define LED_STRIP_PIN        PA8
+
+#define UART1_TX_PIN         PA9
+#define UART2_TX_PIN         PD5
+#define UART3_TX_PIN         PD8
+#define UART4_TX_PIN         PB9
+#define UART6_TX_PIN         PC6
+#define UART7_TX_PIN         PE8
+#define UART8_TX_PIN         PE1
+#define UART1_RX_PIN         PA10
+#define UART2_RX_PIN         PD6
+#define UART3_RX_PIN         PD9
+#define UART4_RX_PIN         PB8
+#define UART6_RX_PIN         PC7
+#define UART7_RX_PIN         PE7
+#define UART8_RX_PIN         PE0
+
+#define I2C1_SCL_PIN         PB6
+#define I2C2_SCL_PIN         PB10
+#define I2C1_SDA_PIN         PB7
+#define I2C2_SDA_PIN         PB11
+
+#define LED0_PIN             PE3
+#define LED1_PIN             PE4
+
+#define SPI1_SCK_PIN         PA5
+#define SPI2_SCK_PIN         PB13
+#define SPI3_SCK_PIN         PB3
+#define SPI4_SCK_PIN         PE12
+#define SPI1_SDI_PIN         PA6
+#define SPI2_SDI_PIN         PB14
+#define SPI3_SDI_PIN         PB4
+#define SPI4_SDI_PIN         PE13
+#define SPI1_SDO_PIN         PD7
+#define SPI2_SDO_PIN         PB15
+#define SPI3_SDO_PIN         PB5
+#define SPI4_SDO_PIN         PE14
+
+#define ADC_VBAT_PIN         PC0
+#define ADC_RSSI_PIN         PC5
+#define ADC_CURR_PIN         PC1
+#define ADC_EXTERNAL1_PIN    PC4
+
+#define PINIO1_PIN           PD10
+#define PINIO2_PIN           PD11
+
+#define FLASH_CS_PIN         PE2
+#define MAX7456_SPI_CS_PIN   PB12
+
+#define GYRO_1_EXTI_PIN      PB2
+#define GYRO_2_EXTI_PIN      PE15
+#define GYRO_1_CS_PIN        PC15
+#define GYRO_2_CS_PIN        PE11
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP(  0, PA15, 1,  0) \
+    TIMER_PIN_MAP(  1, PB0 , 2,  0 ) \
+    TIMER_PIN_MAP(  2, PB1 , 2,  1 ) \
+    TIMER_PIN_MAP(  3, PA0 , 2,  2 ) \
+    TIMER_PIN_MAP(  4, PA1 , 2,  3 ) \
+    TIMER_PIN_MAP(  5, PA2 , 2,  4 ) \
+    TIMER_PIN_MAP(  6, PA3 , 2,  5 ) \
+    TIMER_PIN_MAP(  7, PD12, 1,  6 ) \
+    TIMER_PIN_MAP(  8, PD13, 1,  7 ) \
+    TIMER_PIN_MAP(  9, PD14, 1, -1 ) \
+    TIMER_PIN_MAP( 10, PD15, 1, -1 ) \
+    TIMER_PIN_MAP( 11, PE5 , 1, -1 ) \
+    TIMER_PIN_MAP( 12, PE6 , 1, -1 ) \
+    TIMER_PIN_MAP( 13, PA8 , 1, 10 )
+
+#define ADC1_DMA_OPT        8
+#define ADC3_DMA_OPT        9
+#define TIMUP1_DMA_OPT      0
+#define TIMUP3_DMA_OPT      0
+#define TIMUP4_DMA_OPT      0
+#define TIMUP5_DMA_OPT      0
+
+#define DEFAULT_GYRO_TO_USE GYRO_CONFIG_USE_GYRO_BOTH
+#define MAG_I2C_INSTANCE I2CDEV_1
+#define BARO_I2C_INSTANCE I2CDEV_2
+#define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_FLASH
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE 250
+#define BEEPER_INVERTED
+#define MAX7456_SPI_INSTANCE SPI2
+#define PINIO1_CONFIG 129
+#define PINIO1_BOX 39
+#define PINIO2_BOX 40
```

Comment:
VTX pit mode is not for power control
```suggestion
#define PINIO1_BOX 40
#define PINIO2_BOX 41
```